tests/unit/LimitedRestorationExtension.spec.ts

```diff
@@ -261,16 +262,17 @@ describe('Restoration component - edit page', () => {
   it('loads the entity snapshot into the store', async () => {
     await wrapper.setProps({ appReady: true })
     await flushPromises()
-    const state = store.state.stateModel
+    const state = store.stateModel
 
     // Validate business identifier
     expect(state.tombstone.businessId).toBe('BC1234567')
 
     // Validate Business
     expect(state.businessInformation.legalType).toBe('BEN')
     expect(state.businessInformation.legalName).toBe('1234567 B.C. LTD.')
-    expect(state.businessInformation.stateFiling.restoration.type).toBe('limitedRestoration')
-    expect(state.businessInformation.stateFiling.restoration.parties[0].role).toBe('applicant')
+    // Travis Semple - stateFiling is defined as a string here, not an object. should be fixed.
```

Comment:
Dropping a comment here so I can check this is fixed by final commit :)

Reply:
I think this is out of the scope of implementing Pinia - this is an existing bug

Reply:
This may be confusion between the state filing object and the state filing URL (string).

Yes, this is out of scope. Do you mind creating a bug/techdebt ticket? Thx,